config/crd/bases/core.spinoperator.dev_spinapps.yaml

```diff
@@ -189,7 +189,11 @@ spec:
                     type: object
                 type: object
               components:
-                description: Components to be executed in this group.
+                description: |-
+                  Components of the app to execute.
+
+
+                  If this is not provided all components are executed.
```

Comment:
Why is this changing here? Did we forget to run `make` in a previous PR and this is fixing that?

Reply:
Must be. Maybe there is a way to add a check for this in CI

Reply:
https://github.com/spinkube/spin-operator/issues/342